lib/module.js

```diff
@@ -74,6 +74,22 @@ function Module(id, parent) {
 }
 module.exports = Module;
 
+Object.defineProperty(
+  module.exports,
+  'builtinModules',
+  {
+    enumerable: true,
+    get: function() {
+      return Object.keys(process.binding('natives'))
+        .filter((module) => {
+          return !module.includes('internal/') &&
+            !module.includes('v8/') &&
+            !module.includes('node-inspect/');
```

Comment:
perhaps `startsWith` instead of `includes`?

Reply:
+1 for strictness